File: src/main/java/ru/mydesignstudio/database/metadata/extractor/extractors/constraint/ConstraintModel.java
package ru.mydesignstudio.database.metadata.extractor.extractors.constraint;

import lombok.Data;

@Data
public class ConstraintModel {
  private String constraintType;
  private String schemaName;
  private String constraintName;
  private String tableName;
  private String columns;
  private String rOwner;
  private String rConstraintName;
}

File: extractor-core/src/main/kotlin/ru/mydesignstudio/database/metadata/extractor/extract/result/ConstraintModel.kt
package ru.mydesignstudio.database.metadata.extractor.extract.result

data class ConstraintModel(
        val constraintType: String,
        val schemaName: String,
        val constraintName: String,
        val tableName: String,
        val columns: String,
        val rOwner: String,
        val rConstraintName: String
)